src/gui/mzroll/alignmentdialog.cpp

```diff
@@ -201,17 +201,17 @@ void AlignmentDialog::algoChanged()
     refSampleLabel->setVisible(obiWarp);
 
     auto samples = _mw->getSamples();
-    auto ms2SamplesDetected = false;
+    auto mrmData = false;
     for (const auto sample : samples) {
-        if (sample->ms2ScanCount() > 0) {
-            showInfo("Obi-warp does not work with MS2 scans at the moment.\nWe "
+        if (sample->getSampleType() == mzSample::SampleType::SRM || sample->getSampleType() == mzSample::SampleType::MRM ) {
```

Comment:
This can be made pretty simple. Just check if there are MS1 scans. If yes, alignment can be done on the MS1 scans. If not, a message pops up saying "There are no MS1 scans for alignment". No need for sample types etc.

Reply:
The types were introduced to make the code more readable.  Checking for `SRM/MRM` types makes it clear that there are certain types of sample that are not supported by OBI-WARP, just checking for `ms1Scans`, which I believe is a more technical term, will leave the readers in a more ambiguous state. Let me know if you think otherwise

Reply:
Thanks for the change. We can discuss the error message with the product team and come up with a simpler message for the users if required!